infra/experimental/sanitizers/ExecSan/execSan.cpp

```diff
@@ -60,6 +62,86 @@
 const std::string kTripWire = "/tmp/tripwire";
 // The PID of the root process we're fuzzing.
 pid_t g_root_pid;
+// Assuming the longest pathname is "/bin/bash".
+const int kShellPathnameLength = 10;
+// Assuming the syntax error pattern is
+//   within the first 100 chars of the write buffer.
+const int kErrorMessageLength = 100;
+// Shell injection bug confirmed with /tmp/tripwire.
+const int kShellInjection = 1;
+// Shell corruption bug speculated based on syntax error.
```

Comment:
I'm more familiar with the term "shell injection" than shell corruption, can we try to use this name everywhere.

Reply:
Also "speculated"->"detected"